app/View/Components/UserCard.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\View\Components;
+
+use Closure;
+use App\Models\User;
+use Illuminate\Contracts\View\View;
+use Illuminate\View\Component;
+
+class UserCard extends Component
+{
+    public $birthday;
+    /**
+     * Create a new component instance.
+     */
+    public function __construct(public User $user)
+    {
+        $this->birthday = $this->setBirthday($user->created_at);
```

Comment:
change to reflect the firstChirped names. in the construct method, assign $firstChirp to the user's first chirp, OR null if they have not chirped. then pass the firstChirp into the setFirstChirped method